compiler/rustc_resolve/src/late.rs

```diff
@@ -190,6 +190,7 @@ pub(crate) enum RibKind<'a> {
     ///
     /// The item may reference generic parameters in trivial constant expressions.
     /// All other constants aren't allowed to use generic params at all.
+    // FIXME(generic_consts): Update the docs above.
```

Comment:
maybe the comment doesn't need updating at all. it might actually answer my question regarding (pseudo) `const ADD<const N, const M> = N + M` and why it doesn't throw an "cannot ref gens" error.

Remove this FIXME and add a GitHub comment on the offical PR mentioning this topic.